<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SDKWinForm</class>
 <widget class="QMainWindow" name="SDKWinForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>627</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Xpeccy SDK (alpha)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,10">
    <item>
     <widget class="QWidget" name="butWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QToolButton" name="prjNew">
         <property name="toolTip">
          <string>New project</string>
         </property>
         <property name="text">
          <string>New</string>
         </property>
         <property name="icon">
          <iconset resource="../xpeccy.qrc">
           <normaloff>:/images/book-new.png</normaloff>:/images/book-new.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="prjOpen">
         <property name="toolTip">
          <string>Open project</string>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
         <property name="icon">
          <iconset resource="../xpeccy.qrc">
           <normaloff>:/images/fileopen.png</normaloff>:/images/fileopen.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="prjSave">
         <property name="toolTip">
          <string>Save project</string>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
         <property name="icon">
          <iconset resource="../xpeccy.qrc">
           <normaloff>:/images/floppy.png</normaloff>:/images/floppy.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="prjBuild">
         <property name="text">
          <string>Build</string>
         </property>
         <property name="icon">
          <iconset resource="../xpeccy.qrc">
           <normaloff>:/images/other.png</normaloff>:/images/other.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="prjtree">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="10,0">
      <item>
       <widget class="QTextEdit" name="docedit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans Mono</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="console"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>883</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../xpeccy.qrc"/>
 </resources>
 <connections/>
</ui>
